glib/Gir_GObject.toml

```
@@ -103,6 +103,44 @@ concurrency = "send+sync"
     name = "source"
     generate = ["notify"]
 
+[[object]]
+name = "GObject.SignalGroup"
+status = "generate"
+final_type = true
+concurrency = "send+sync"
```

```suggestion
concurrency = "send+sync"
version = "2.74"
```

Probably? It's unusable from the bindings without the `connect_closure()` function